lib/resty/mlcache.lua

```diff
@@ -765,7 +765,8 @@ function _M:get(key, opts, cb, ...)
 
     local pok, perr, err, new_ttl = xpcall(cb, traceback, ...)
     if not pok then
-        return unlock_and_ret(lock, nil, "callback threw an error: " .. perr)
+        return unlock_and_ret(lock, nil,
+            "callback threw an error: " .. tostring(perr))
```

Comment:
style: this module aligns newline arguments to the first argument from the previous line, like so:
```lua
return unlock_and_ret(lock, nil,
                      "callback threw an error: " .. tostring(perr))
```